readme.md

```diff
@@ -236,7 +236,9 @@ Type: `string | Buffer | stream.Readable` or [`form-data` instance](https://gith
 
 **Note #4:** This option is not enumerable and will not be merged with the instance defaults.
 
-The `content-length` header will be automatically set if `body` is a `string` / `Buffer` / `fs.createReadStream` instance / [`form-data` instance](https://github.com/form-data/form-data), and `content-length` and `transfer-encoding` are not manually set in `options.headers`.
+The `content-length` header will be automatically set if `body` is a `string` / `Buffer` / [`form-data` instance](https://github.com/form-data/form-data), and `content-length` and `transfer-encoding` are not manually set in `options.headers`.
+
+The `content-length` is no more automatically set when `body` is a `fs.createReadStream`.
```

Comment:
I'd start the sentence with `Since Got 12`

Reply:
You need to update the doc comments too.

Reply:
> I'd start the sentence with `Since Got 12`

I choose to avoid version related comments because I'm worried the README could became became a wall of
`Since Got ## [...]`

One option could be inserting it in Got 12 and removing it in Got 13, let me know what you think about this.

Reply:
I think it's useful to show when it was changed. A couple. of major versions later, we can remove the mention of the version.